sop-website/docs/Data-Upload-Update/Data-Uploads.mdx

```diff
@@ -0,0 +1,418 @@
+
+4. Configure /etc/hosts
+You need to modify the `/etc/hosts` file on the system from which you will be running the upload tool.
+
+You will need root access to edit this file.  Add the following to the file:
+```
+172.203.106.139             choruspilotstorage.blob.core.windows.net
+```
+
+If this is not configured, you may see error in AZ CLI like so:
+
+```
+The request may be blocked by network rules of storage account. Please check network rule set using 'az storage account show -n accountname --query networkRuleSet'.
+If you want to change the default action to apply when no rule matches, please use 'az storage account update'.
+```
+
+And with the built-in azure python library:
+```
+HttpResponseError: Operation returned an invalid status 'This request is not authorized to perform this operation.'
+ErrorCode:AuthorizationFailure
+```
+
+
+On windows, the `/etc/hosts` file is instead `C:\Windows\system32\drivers\etc\hosts`.   Administrator privilege is needed to edit this file.
+
+5. AZ CLI installation (only when using `chorus-upload` generated azcli scripts)
+You can configure the tool to use AZ CLI to upload files to the CHoRUS central cloud, or alternatively use the built in azure library for upload.   If you will be using AZ CLI, please install AZ CLI according to Microsoft instructions:
+
+[Install Azure CLI](https://learn.microsoft.com/en-us/cli/azure/install-azure-cli)
+
+
+6. Setting environment variables
+Also, please make sure that the following environment variable is set.  It is recommended that they are set in Linux .profiles file or as Windows user environment variable.
+
+Windows
+```
+set AZURE_CLI_DISABLE_CONNECTION_VERIFICATION 1
+```
+
+Linux
+```
+export AZURE_CLI_DISABLE_CONNECTION_VERIFICATION=1
+```
+
+
+
+### Getting Azure credential
+
+For users with CHoRUS cloud storage access via the Azure Portal, please generate a sas token for your container.  If you do not have access, please reach out to a member of the CHoRUS central cloud team.
+
+From the Azure Portal, navigate to `Storage Account` / `Containers`, and select your DGS container.  Please make note of the account name (should be `choruspilotstorage`) and the container name (should be a short name for your institution).  In the left menu, select `Settings` / `Shared access tokens`.  Create a new SAS token with `Read`, `Add`, `Create`, `Write`, and `List` enabled, and optionally `Delete` if you intend to use the same sas token for deletion later.  The expiration of the SAS token is recommended to be 1 month from the creation date.  Copy the SAS token string and save it in a secure location.  The SAS token will be used by the `chorus-upload` tool. 
+
+If you are transferring files from a cloud account to CHoRUS, please refer to you institution's documentation to retrieve credentials for other storage clouds.  For a list of supported authentication mechanisms for each tested cloud providers, please see the `config.toml.template` file.
```

Comment:
Typo:

If you are transferring files from a cloud account to CHoRUS, please refer to you institution's documentation to retrieve credentials for other storage clouds.
-> 
If you are transferring files from a cloud account to CHoRUS, please refer to your institution's documentation to retrieve credentials for other storage clouds.

Reply:
fixed.  Thanks!